src/selectors/getSketchParams.js

```diff
@@ -8,7 +8,7 @@ const getSingle = (state, sketchId) => {
 
   sketchParams.forEach((id) => {
     const param = state.nodes[id]
-    if (param.min && param.max) {
+    if (param.min !== undefined && param.max !== undefined) {
```

Comment:
Min/max wasn't working at all if one of the values was set to zero, as it being seen as false. This is a good example of why tests are useful (see above test added), especially for complicated selectors. I often write the test first in these cases.